geth/whisper/request_historic_messages.go

```diff
@@ -0,0 +1,151 @@
+package whisper
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/binary"
+	"fmt"
+	"github.com/ethereum/go-ethereum/p2p/discover"
+	"github.com/ethereum/go-ethereum/whisper/whisperv5"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/rpc"
+	"time"
+)
+
+//RequestHistoricMessagesHandler returns rpc which send p2p requestMessagesRequest for expired messages.
```

Comment:
Fix grammar here, please. It should be `// RequestHistoricMessagesHandler returns an RPC handler which sends a p2p request for historic messages`.